packages/test/test-end-to-end-tests/src/test/gc/gcContainerRuntimeExpiry.spec.ts

```diff
@@ -47,22 +49,46 @@ import { TestDataObject } from "./mockSummarizerClient";
     );
 
     let container1: IContainer;
-
     const createContainer = async (): Promise<IContainer> => provider.createContainer(runtimeFactory);
+    const loadContainer = async (): Promise<IContainer> => provider.loadContainer(runtimeFactory);
+    let clock: SinonFakeTimers;
 
-    beforeEach(async () => {
-        provider = getTestObjectProvider();
+    before(() => {
+        clock = useFakeTimers();
     });
 
-    it("Container should be closed with a ClientSessionExpired error after the gcSessionExpiryTime is up", async () => {
+    afterEach(() => {
+        clock.reset();
+    });
+
+    after(() => {
+        clock.restore();
+    });
+
+    beforeEach(async () => {
+        provider = getTestObjectProvider();
         container1 = await createContainer();
         container1.on("closed", (error) => {
             assert.strictEqual(error?.errorType, "clientSessionExpiredError");
         });
+    });
 
+    it("Container should be closed with a ClientSessionExpired error after the gcSessionExpiryTime is up", async () => {
         await provider.ensureSynchronized();
-        const delay = async (ms: number) => new Promise((res) => setTimeout(res, ms));
-        await delay(100);
-        assert(container1.closed === true, "Container should be closed");
+        clock.tick(timeoutMs - 1);
+        assert(container1.closed === false, "Container1 should not be closed");
+        clock.tick(1);
+        assert(container1.closed, "Container should be closed");
```

Comment:
nit: Container1. Also, in the asserts, could you add why the container should be closed / not closed.